From 0bcdf5ffef2d7e65d5331e7a0c8f583108fe7e25 Mon Sep 17 00:00:00 2001
From: Mathias Biilmann Christensen <mbc@mbcbook-2.local>
Date: Tue, 16 Mar 2010 12:26:14 +0100
Subject: [PATCH] Fix for missing dependency in ActionDispatch::Integration

When running cucumber features from a new rails 3 app requiring
'cucumber/rails/world' would raise:

uninitialized constant ActionDispatch::Integration::Session::Test (NameError)

Fixed by requiring 'test/unit/assertions' in action_dispatch/integration
---
 .../lib/action_dispatch/testing/integration.rb     |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/actionpack/lib/action_dispatch/testing/integration.rb b/actionpack/lib/action_dispatch/testing/integration.rb
index 31067e5..305c2bf 100644
--- a/actionpack/lib/action_dispatch/testing/integration.rb
+++ b/actionpack/lib/action_dispatch/testing/integration.rb
@@ -2,6 +2,7 @@ require 'stringio'
 require 'uri'
 require 'active_support/core_ext/object/singleton_class'
 require 'rack/test'
+require 'test/unit/assertions'
 
 module ActionDispatch
   module Integration #:nodoc:
-- 
1.6.3.1

